src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php

```diff
@@ -589,13 +609,20 @@ public function template($index = 9999999999999)
             'list_subscription' => __('List Selection', 'mailoptin'),
             'recaptcha_v2'      => __('reCAPTCHA v2', 'mailoptin'),
             'recaptcha_v3'      => __('reCAPTCHA v3', 'mailoptin'),
+            'country'            => __('Country', 'mailoptin'),
         ];
 
         $widget_title = sprintf(__('Field %s', 'mailoptin'), '#' . ($index + 1));
         if (isset($this->saved_values[$index]['placeholder'])) {
             $widget_title = $this->saved_values[$index]['placeholder'];
         }
 
+        $list_country_name_select_type = [
+            ''  =>   __('Select One..', 'mailoptin'),
```

Comment:
Remove this. Let it always default to alpha-2.

Reply:
Okay